microprot/scripts/processing.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__ = "Tomasz Kosciolek"
+__version__ = "1.01b"
+__last_update__ = "12/19/2016"
```

Comment:
I would remove this and only use git log to track the latest update. It would be a pain in the ass to keep this info up to date, i.e. useful.